example/default.css

```diff
@@ -0,0 +1 @@
+../themes/default/default.css
```

Comment:
> In a next iteration we'll really add the file from the build.

Ah yes, we will need to add some css build to the dev task to do so. To be able to modify css live.

Dunno what webpack can bring here.